### ISSUE TITLE
ListChangeNotifier doesn't notify listeners on remove

### ISSUE THREAD

**Issue by lenlo** [flutter-institute/collection_providers#3]
If you remove an item from a ListChangeNotifier using the [mixed-in] `remove` method, the listeners won't be notified of the change.

ListChangeNotifier needs to have its own `remove` cover method that calls `super.remove(item)` and then proceeds to call `notifyListeners()` if the item got removed.

Something like this:

```
  @override
  bool remove(Object? value) {
    assert(_debugAssertNotDisposed());
    final result = super.remove(value);
    if (result)
      notifyListeners();
    return result;
  }

```

Thanks!

**Comment by killermonk** [comment]
Fixed in 1cec36b3272fddebba5d69aa316983dd6d3045ae
Just published version 0.2.2 with these changes. Should be live in pub soon.

**Comment by killermonk** [comment]
Should I have made a pull request for these changes? Yes.
Did I make a pull request for these changes instead of just pushing straight to master? No.
